docs/configuration/configuration_old.rst

```diff
@@ -249,14 +249,23 @@ The spectrum section defines the
 
     spectrum:
         start : 500 angstrom
-        end : 20000 angstrom
-        bins : 1000
-        sn_distance : lum_density
-        #sn_distance : 10 Mpc
-
-Start and end are given as Quantities with units. If they are given in frequency space they are switched around if
-necessary. The number of bins is just an integer. Finally the ``sn_distance`` can either be a distance or the special
-parameter ``lum_density`` which sets the distance to :math:`\sqrt{\frac{1}{4 \pi}}` to calculate the luminosity density.
+        stop : 20000 angstrom
+        num : 1000
+        method: virtual
+
+Start and end are given as Quantities with units. If they are given in
+frequency space they are switched around if necessary. The number of bins is
+just an integer. Finally, the method option selects the final spectral synthesis mode. Currently, there are three options:
```

Comment:
Well, currently this may be true. But eventually, some of these will be mutually exclusive. For example, when the Formal integral method is selected, the virtual packet scheme will be deactivated.